### DISCUSSION TITLE
Need details on plan for replication from on-premise (via WiltonDB) to AWS Aurora PostgreSQL cloud

### DISCUSSION THREAD

**Discussion by psrinivasa2** [babelfish-for-postgresql/babelfish-for-postgresql#31] [Q&A]
Hi,
Need to know details on plan for replication from on-premise (via WiltonDB) to AWS Aurora PostgreSQL cloud. We mainly need to understand below specific query?

1) How the replication planned is it WiltonDB babelfish to AWS Aurora babelfsh [OR] WiltonDB postgreSQL to AWS Aurora PostgreSQL directly?

2) When can we expect this change to be completed?

**Comment by suprio-amzn** [comment]
@psrinivasa2: Please ask support for features in WiltonDB in that project's discussion/issues forum.
